packages/trpc/server/routers/viewer/bookings/editLocation.handler.ts

```diff
@@ -118,6 +118,10 @@ export const editLocationHandler = async ({ ctx, input }: EditLocationOptions) =
         },
         data: {
           location,
+          metadata: {
+            ...(typeof booking.metadata === "object" ? booking.metadata : {}),
+            videoCallUrl: location,
```

Comment:
What if location is a phone number or address? We then would store it as videoCallUrl in metadata.

Reply:
Here is updated booking metadata:

<img width="428" alt="Screenshot 2024-06-21 at 15 05 27" src="https://github.com/calcom/cal.com/assets/42170848/0330f3f4-14b1-49f0-86e2-42989977bce5">

I think we need to update `videoCallUrl` only if its a link or cal daily video.

Reply:
however, if the location changes from link to phone, then we need to clear metadata.

Reply:
Fixed it. Actually, when the location is a link or a static video app, we only set the videoCallUrl inside the metadata.